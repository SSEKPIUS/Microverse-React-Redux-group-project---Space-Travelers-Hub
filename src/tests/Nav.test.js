import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import store from '../Redux/configureStore';
import NavBar from '../components/Nav';

describe('Test snapshot component of Navbar', () => {
  it('Renders correctly of nav bar', () => {
    const tree = render(
      <BrowserRouter>
        <Provider store={store}>
          <NavBar />
        </Provider>
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
