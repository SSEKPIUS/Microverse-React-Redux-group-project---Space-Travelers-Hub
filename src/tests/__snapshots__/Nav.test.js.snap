// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot component of Navbar Renders correctly of nav bar 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav>
        <div
          class="logo"
        >
           
          <span
            class="icon-button"
          >
            <img
              alt="logo"
              src="logo.png"
              style="width: 3.5em; height: 3.5em;"
            />
          </span>
          <span
            class="nav-brand"
          >
            Space Travelers' Hub
          </span>
        </div>
        <ul
          class="nav-links"
        >
          <li>
            <a
              class="nav-link "
              href="/rockets"
            >
              Rockets
            </a>
          </li>
          <li>
            <a
              class="nav-link "
              href="/dragons"
            >
              Dragons
            </a>
          </li>
          <li>
            <a
              class="nav-link "
              href="/missions"
            >
              Missions
            </a>
          </li>
          <li>
            <a
              class="nav-link "
              href="/profile"
            >
              My Profile
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav>
      <div
        class="logo"
      >
         
        <span
          class="icon-button"
        >
          <img
            alt="logo"
            src="logo.png"
            style="width: 3.5em; height: 3.5em;"
          />
        </span>
        <span
          class="nav-brand"
        >
          Space Travelers' Hub
        </span>
      </div>
      <ul
        class="nav-links"
      >
        <li>
          <a
            class="nav-link "
            href="/rockets"
          >
            Rockets
          </a>
        </li>
        <li>
          <a
            class="nav-link "
            href="/dragons"
          >
            Dragons
          </a>
        </li>
        <li>
          <a
            class="nav-link "
            href="/missions"
          >
            Missions
          </a>
        </li>
        <li>
          <a
            class="nav-link "
            href="/profile"
          >
            My Profile
          </a>
        </li>
      </ul>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
